.travis.yml

```diff
@@ -0,0 +1,4 @@
+language: ruby
+sudo: false
```

Comment:
Ty... had an issue w/ engine cart and that right off the bat. Maybe when it is warmed up? :fire: 

https://travis-ci.org/sul-dlss/wfs_rails/builds/95651031#L197